governance/README.md

```diff
@@ -1,3 +1,45 @@
 # Governance
 
-Placeholder for the Governance subproject [proposed here](https://github.com/cncf/sig-contributor-strategy/issues/7).
+The Governance subproject of Contributor Strategy aims to assist all CNCF projects with implementing good open source governance practices.  If you are looking for the governance documents of SIG-Contributor-Strategy itself, please read CONTRIBUTING.md.
+
+If you are interested in helping with this project, please speak up on #sig-contributor-strategy in CNCF Slack, or on the mailing list.
+
+## Goals of the Subproject
+
+1. Create, share, and maintain documentation on implementing good open source governance for CNCF projects.
+2. Coach and advise CNCF projects, and prospective projects, leaders in implementing and improving governance.
+3. Advise the CNCF TOC on updating governance criteria for acceptance and graduation.
+
+## Scope
+
+The Governance subproject is focused on, and restricted to, matters related to CNCF project governance, defined as project participation roles, rules, and procedures.  For more information on what governance is, please see governance/documents/definition.md. As examples, this would include issues like:
```

Comment:
Should "please see governance/documents/definition.md" link to somewhere or is that a placeholder?

Reply:
Yeah, it should link to README.md in the documentation subfolder, will fix, thanks!

Reply:
Link fixed.